1-js/03-code-quality/04-ninja-code/article.md

```diff
@@ -1,240 +1,238 @@
 # Ninja code
 
 
-```quote author="Confucius (Analects)"
-Learning without thought is labor lost; thought without learning is perilous.
+```quote author="Khổng Tử (Luận ngữ)"
+Học mà không suy nghĩ thì vô ích; suy nghĩ mà không học thì nguy hiểm.
 ```
 
-Programmer ninjas of the past used these tricks to sharpen the mind of code maintainers.
+Các lập trình viên ninja trong quá khứ đã sử dụng những thủ thuật này để mài giũa tâm trí của những người bảo trì mã.
 
-Code review gurus look for them in test tasks.
+Các chuyên gia đánh giá mã tìm kiếm chúng trong các nhiệm vụ kiểm tra.
```

Comment:
Mình nghĩ nên giữ lại thuật ngữ `Code review` thay vì dịch nó là Đánh giá mã
Tương tự với test tasks
How do you think @tunggnu ?

Reply:
- @henryonsoftware `Code review` dịch thành `đánh giá mã` cũng được, không quen thuộc nhưng cũng không tối nghĩa. Ngoài ra phần hướng dẫn trong README của repo chỉ lưu ý về việc dịch các thuật ngữ chuyên môn như `resolved promise`, `slash`, `regexp` thì cần cân nhắc kỹ, còn `code review` không phải thuật ngữ nên tùy dịch giả, miễn không tối nghĩa hay từ ngữ lủng củng là được.
- Tương tự với `test tasks`, dịch cũng được nhưng mình thấy `task` dịch thành `tác vụ` thì hợp lý hơn `nhiệm vụ`, và `test` dịch là `kiểm thử` thì quen thuộc hơn (nhiều sách IT tiếng Việt dịch như vậy).